deploy/scripts/src/authentication/test.ts

```diff
@@ -445,25 +488,10 @@ export function signIn(): void {
             }),
           { isStatusCode302 }
         )
-        // 02_OIDCCall
-        res = timeGroup(groups[9].split('::')[1], () => http.get(res.headers.Location, { redirects: 0 }), {
-          isStatusCode302
-        })
-
-        acceptNewTerms = res.headers.Location.includes('updated-terms-and-conditions')
-        if (acceptNewTerms) {
-          // 03_AuthAcceptTerms
-          res = timeGroup(groups[10].split('::')[1], () => http.get(res.headers.Location), {
-            isStatusCode200,
-            ...pageContentCheck('terms of use update')
-          })
-        } else {
-          // 03_RPStub
-          res = timeGroup(groups[11].split('::')[1], () => http.get(res.headers.Location), {
-            isStatusCode200,
-            ...pageContentCheck(userData.email.toLowerCase())
-          })
-        }
+        const headerLocation = res.headers.Location
+
+        // 02_OIDCCall, 03_AuthAcceptTerms, 03_RPStub
+        MFAEnterOTP(headerLocation, groupMap.signIn, 9, 10, 11)
```

Comment:
As this is only called in the `signIn` function this can just be moved around within the structure of this scenario. It can just be moved to after the switch statement with the relevant Group names handled